lib/src/dispatch_settings.dart

```diff
@@ -0,0 +1,128 @@
+bool _whereNotOlderThanADay(Map<String, String> action) =>
+    DispatchSettings.whereNotOlderThan(
+      const Duration(
+        hours: 23,
+        minutes: 59,
+        seconds: 59,
+      ),
+    )(action);
+
+bool _notOlderThan(
+  Map<String, String> action,
+  DateTime now,
+  Duration duration,
+) {
+  final date = action['cdt']!;
+  return now.difference(DateTime.parse(date)) <= duration;
+}
+
+bool _takeAll(Map<String, String> action) => true;
```

Comment:
Using 
```dart
  /// Takes all actions.
  static const PersistenceFilter takeAll = (Map<String, String> action) => true;
```
results in
```
error - Const variables must be initialized with a constant value - lib\src\dispatch_settings.dart:80:44 - const_initialized_with_non_constant_value
```
changing the `const` to `final` would conflict with the constructor `DispatchSettings.persistent` beeing `const`, and more importantly give the linter warning 
```
Use a function declaration rather than a variable assignment to bind a function to a name.
Try rewriting the closure assignment as a function declaration. (dart prefer_function_declarations_over_variables)
```
To fix all this we could of course use
```dart
  /// Takes all actions.
  static bool takeAll(Map<String, String> action) => true;
```
But then we lose the `PersistenceFilter` typing which I thought might be nice to have for documentational purpose, since in my opinion it is much more clear having a member of type `PersistenceFilter` then simply having a `bool Function(Map<String, String> action)` in the class. Both are of course equivalent but still I think one gives away a bit more information on what to do with the thing.